test/unit/rules/role-has-required-attributes-test.js

```diff
@@ -0,0 +1,128 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'role-has-required-attributes',
+
+  config: true,
+
+  good: [
```

Comment:
Remember to test on an existing codebase to ensure no crashes.

Reply:
FWIW, just did a quick test on our codebase. No crashes here 👍.

Reply:
Great, thanks so much @bertdeblock !